File: materiale/programmazione/E0_primo/prova.cs
public static class Prova
{
    double calcRadice(int numero)
    {
        return Math.Sqrt(numero);
    }

    public static void Main()
    {
        Console.ReadLine("Inserisci un numero per calcolare la radice quadrata: ");
        int radice = Convert.ToInt32(Console.ReadLine());
        double radiceCalcolare = calcRadice(radice);

        Console.WriteLine("La radice quadrata di {0} è: {1}", radice, radiceCalcolare);
    }
}

File: materiale/programmazione/E0_primo/prova.c
#include <stdio.h>
#include <math.h>

int radiceDaCalc = 3;

double radiceCalcolata;


double calcoloRadiceQuad()
{
    double res = sqrt(radiceDaCalc);
    return res;
}

int main()
{
    radiceCalcolata = calcoloRadiceQuad();
    printf("La radice quadrata di %d è: %f", radiceDaCalc, radiceCalcolata);
    return 0;
}